src/clients/Governance.ts

```diff
@@ -363,4 +366,17 @@ export class Governance extends API {
     )
     return response.data
   }
+
+  async getTransparencyBudgets() {
+    const response = await this.fetch<ApiResponse<TransparencyBudget[]>>(`/budget/fetch`, this.options().method('GET'))
+    return response.data
+  }
+
+  async updateGovernanceBudgets() {
+    const response = await this.fetch<ApiResponse<QuarterBudgetAttributes[]>>(
+      `/budget/update`,
+      this.options().method('GET').authorization({ sign: true })
```

Comment:
Since this endpoint modifies the database, I would use a POST or PATCH instead of a GET